### PR TITLE
[RU] Add `Beatmapping/Mapping techniques/Flow`

### PR DESCRIPTION
**Pull request by TicClick**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Beatmapping/Mapping_techniques/Flow/ru.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+tags:
+  - linear flow
+  - oval flow
+  - circular flow
+  - флоу
+---
+
+# Флоу
+
+Флоу (англ. *flow*, букв. *течение*) — концепт в маппинге, описывающий естественное и интуитивное движение курсора во время игры и относящийся преимущественно к [osu!](/wiki/Game_mode/osu!), поскольку только там есть двумерное игровое поле. Флоу создаётся с помощью плавных, переходящих друг в друга [паттернов](/wiki/Beatmap/Pattern) и взаимодополняющих форм слайдеров, благодаря которым игрок направляется в нужную игровую область.
```
Comment (UristMcKobold) [inline wiki/Beatmapping/Mapping_techniques/Flow/ru.md:11]:
```suggestion
Флоу (англ. *flow*, букв. *течение*) — концепт в маппинге, описывающий естественное и интуитивное движение курсора во время игры и относящийся преимущественно к [osu!](/wiki/Game_mode/osu!), поскольку только там есть двумерное игровое поле. Флоу создаётся с помощью плавных, переходящих друг в друга [паттернов](/wiki/Beatmap/Pattern) и взаимодополняющих форм слайдеров, благодаря которым курсор игрока направляется в нужную часть экрана.
```

File: wiki/Beatmapping/Mapping_techniques/Flow/ru.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Флоу считается **субъективным** и часто спорным явлением, поскольку разные игроки и мапперы по-разному понимают и ощущают движение. В зависимости от [устройства ввода](/wiki/Gameplay/Input_device), — например, [планшета](/wiki/Gameplay/Input_device/Graphics_tablet) или [мыши](/wiki/Gameplay/Input_device/Mouse), — а также навыков игрока, флоу может как серьёзно влиять на впечатления от игры, так и не привносить никакой разницы.
```
Comment (UristMcKobold) [inline wiki/Beatmapping/Mapping_techniques/Flow/ru.md:13]:
```suggestion
Флоу считается **субъективным** и часто спорным понятием, поскольку разные игроки и мапперы по-разному видят и воспринимают движение курсора. В зависимости от [устройства ввода](/wiki/Gameplay/Input_device), — например, [планшета](/wiki/Gameplay/Input_device/Graphics_tablet) или [мыши](/wiki/Gameplay/Input_device/Mouse), — а также навыков игрока, флоу может как серьёзно влиять на впечатления от игры, так и не привносить никакой разницы.
```

File: wiki/Beatmapping/Mapping_techniques/Flow/ru.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+### Визуальные подсказки
+
+В понятие флоу также входит применение различных видов слайдеров, которые намекают на характер движения, ожидаемый от игрока. Поскольку слайдер необязательно проводить идеально по видимой траектории (достаточно собрать только [слайдер-тики](/wiki/Gameplay/Hit_object/Slider/Slider_tick) и начало с концом), то ему можно придать самые разнообразные формы, визуально указывающие, куда «нужно» двигаться. Слайдеры, выполненные таким образом, дополняют предшествующую им траекторию курсора и «подталкивают» игроков в определённом направлении, порой усиливая движущий импульс.
+
+## Применение
+
+Флоу применяется для отражения в карте различных особенностей музыки. Например, с помощью плавного флоу, контрастирующего с неестественным движением, можно подчеркнуть интенсивность ключевых моментов песни. Повторяющиеся паттерны и движения, в свою очередь, часто соотносятся с повторяющимися фрагментами музыки.
```
Comment (UristMcKobold) [inline wiki/Beatmapping/Mapping_techniques/Flow/ru.md:37]:
```suggestion
Флоу применяется для отражения в карте различных особенностей музыки. Например, с помощью плавного флоу, контрастирующего с неестественным движением, можно подчеркнуть интенсивность ключевых моментов песни. Повторяющиеся паттерны и такие же движения курсором, в свою очередь, часто соотносятся с повторяющимися фрагментами музыки.
```

File: wiki/Beatmapping/Mapping_techniques/Flow/ru.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## См. также
+
+- [Analyzing Flow in Beatmaps](https://osu.ppy.sh/community/forums/topics/86329), руководство от ::{ flag=US }:: [Charles445](https://osu.ppy.sh/users/85000)
+- [Mapping Analysis #1 - Flow](https://osu.ppy.sh/home/news/2013-09-15-mapping-analysis-1-flow), новость от ::{ flag=FR }:: [Shiro](https://osu.ppy.sh/users/113005)
+- [flow + circles](https://www.youtube.com/watch?v=ig7nTSbWwVc) и [flow + sliders](https://www.youtube.com/watch?v=GYGatwQszwQ), эпизоды из цикла видео [osu!mapping](/wiki/Community/Video_series/osu!mapping) от ::{ flag=US }:: [pishifat](https://osu.ppy.sh/users/3178418)
+
+## Примечания
+
+[^circular-flow]: Слишком долгое круговое движение считается неоригинальным, а также не очень удобным среди игроков на мыши, поскольку приводит к дрифту.
```
Comment (UristMcKobold) [inline wiki/Beatmapping/Mapping_techniques/Flow/ru.md:49]:
```suggestion
[^circular-flow]: Слишком долгое круговое движение считается неоригинальным, а также не очень удобным среди игроков на мыши, поскольку приводит к маусдрифту.
```

Comment (TicClick) [reply wiki/Beatmapping/Mapping_techniques/Flow/ru.md:49]: лучше просто «к её дрифту»

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**